post-processor/import/post-processor.go

```diff
@@ -50,6 +50,7 @@ type Config struct {
 	Format          string            `mapstructure:"format"`
 	Architecture    string            `mapstructure:"architecture"`
 	BootMode        string            `mapstructure:"boot_mode"`
+	TpmSupport      string            `mapstructure:"tpm_support"`
```

Comment:
This is added to the post-processor here, but I don't see the related step being updated, is this voluntary?